src/material-experimental/mdc-radio/radio.scss

```diff
@@ -1,7 +1,21 @@
 @import '@material/radio/mixins.import';
+@import '@material/radio/variables.import';
 @import '@material/form-field/mixins.import';
 @import '../mdc-helpers/mdc-helpers';
 
-
 @include mdc-radio-without-ripple($query: $mat-base-styles-query);
 @include mdc-form-field-core-styles($query: $mat-base-styles-query);
+
+.mat-mdc-radio-button .mat-radio-ripple {
```

Comment:
Add a comment here that explains why this extra css is necessary

Reply:
Added comments